### PR TITLE
Automated plugin release

### PR DESCRIPTION
**Pull request by a-st**
Implemented automated plugin release

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -47,7 +56,7 @@
     <properties>
         <revision>3.21</revision>
```
Comment (NotMyFault) [inline pom.xml:57]:
```suggestion
        <changelist>999999-SNAPSHOT</changelist>
```

**Review by NotMyFault — CHANGES_REQUESTED** [review]
```diff
- <version>${revision}${changelist}</version>
+ <version>${changelist}</version>
```
A couple of things need to be changed, but otherwise lgtm

File: pom.xml (hunk 1 above)
Comment (a-st) [reply pom.xml]: Thanks @NotMyFault , I applied your suggestions

Comment (NotMyFault) [reply pom.xml]: Thanks!

**Pull request merged**